src/App.tsx

```diff
@@ -15,81 +16,132 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
 enum SortType {
   NONE,
   ALPHABET,
   LENGTH,
 }
 
-type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
-};
-
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+): string[] {
+  let visibleGoods = [...goods];
```

Comment:
```suggestion
  const visibleGoods = [...goods];
```

Reply:
![image](https://github.com/mate-academy/react_list-of-goods/assets/114029531/cbcaa11d-8584-445b-952f-5a7edb05857b)

Reply:
you're reassigning the same value, i see

Reply:
Do you have another way? ... show me.

Reply:
![image](https://github.com/mate-academy/react_list-of-goods/assets/55200530/4382b4d7-ca9d-43dd-b742-e15fe99e8694)
still not resolved

Reply:
What about RESET?